### DISCUSSION TITLE
Mount external S3 in Nhost cloud

### DISCUSSION THREAD

**Discussion by dancixx** [nhost/hasura-storage#223] [Q&A]
Hi guys,

is it possible to mount for example Digital Ocean S3 as external storage in Nhost cloud?

**Comment by dbarrosop** [comment]
We currently don't allow it on Free/Pro projects. Just curious, What is the use case?

**Reply by dancixx** [reply]
@dbarrosop I will be honest. I really like nhost production thats the reason why I am using the cloud service instead of the self hosted version, but sometimes I have traffic heavy projects and thats the reason behind my question. Mostly I am using Digitalocean as S3 service because it is much less cost.

**Reply by dbarrosop** [reply]
Thanks.

> but sometimes I have traffic heavy projects and thats the reason behind my question

How much are we talking about if you don't mind sharing? Ballpark number is sufficient (are we talking terabytes of storage and a hundreds of terabytes of outbound transfer a month, orders of magnitude lower/higher?)

> Mostly I am using Digitalocean as S3 service because it is much less cost

But you are not paying for S3 if you are using our cloud. In any case, in a heavy-traffic project you'd actually make the solution more expensive by using DO as your S3-compatible storage as you'd have to pay for the egress between DO->nhost and then for the egress between nhost->client
